css-four-column-layout/styles.css

```diff
@@ -0,0 +1,50 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+}
+
+h2 {
+  text-align: center;
+  font-weight: bold;
+  font-family: Barlow, sans-serif;
+  position: relative;
+  left: 30px;
+}
+
+h3 {
+  font-weight: bold;
+  font-family: Barlow, sans-serif;
+}
+
+p {
+  padding: 20px;
+}
+
+.column-1 {
+  flex-basis: 25%;
+}
+
+.column-2 {
+  text-align: center;
+}
```

Comment:
`column` classes should only ever have `flex-basis` applied to them, nothing else. Anything else should be on another class you can add to the element in addition to the `column` class.

Additionally, `column` class are meant to be named after their size. It might be a good idea to go back and review `css-layout-classes` in full, you seem to have a pretty fundamental misunderstanding of the system.

Reply:
will do 🫡